pybliometrics/scopus/utils/parse_content.py

```diff
@@ -92,9 +93,10 @@ def get_link(dct, idx, path=['coredata', 'link']):
         return None
 
 
-def html_unescape(s:str): 
-        from html import unescape
-        return unescape(s) if s else None
+def html_unescape(s: str):
+    """Convert all named and numeric character references in the
```

Comment:
One line is enough. Something like
"Convert to Unicode characters if possible."